src/thread.rs

```diff
@@ -242,7 +242,20 @@ impl SBThread {
     }
 
     #[allow(missing_docs)]
-    pub fn step_into(
+    pub fn step_into(&self, stop_other_threads: RunMode) -> Result<(), SBError> {
+        let error = SBError::default();
+        unsafe {
+            sys::SBThreadStepInto(self.raw, stop_other_threads);
+        }
+        if error.is_success() {
+            Ok(())
+        } else {
+            Err(error)
+        }
+    }
+
+    #[allow(missing_docs)]
+    pub fn step_into3(
```

Comment:
Your other suggestion for a name is better. What is the target name parameter actually for?

Reply:
Quoting from https://lldb.llvm.org/python_api/lldb.SBThread.html#lldb.SBThread.StepInto:

> Step the current thread from the current source line to the line given by end_line, stopping if the thread steps into the function given by target_name. If target_name is None, then stepping will stop in any of the places we would normally stop.

I had trouble getting this to work with default parameters. That's why I added the simple version. Maybe `until` and `target_name` should be `Option<>` and the method then calls the correct low-level method based on the provided params? But what do you call if only `until` is provided?